server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/metrics/TopHitsIT.java

```diff
@@ -110,7 +118,7 @@ public static String randomExecutionHint() {
 
     @Override
     public void setupSuiteScopeCluster() throws Exception {
-        assertAcked(prepareCreate("idx").setMapping(TERMS_AGGS_FIELD, "type=keyword"));
+        assertAcked(prepareCreate("idx").setMapping(TERMS_AGGS_FIELD, "type=keyword", "text", "type=text,store=true"));
```

Comment:
we used to ask for this is `testFetchFeatures`, without ever checking that it was returned. It was not, because the field was not stored. I added the corresponding assertion below as well.